### PR TITLE
docs: workaround for old opset versions

### PR DESCRIPTION
**Pull request by SuperSecureHuman**
Model version conversion with onnx's python API for supporting some models.

Tested with 

mobilenetv2-7
resnet101-v1-7.onnx
resnet50-caffe2-v1-6.onnx

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,35 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:
...
+<summary>Example</summary>
+
+```python
+import onnx
+from onnx import version_converter
+import torch
+from onnx2torch import convert
+
+# Load the ONNX model
+model = onnx.load("model.onnx")
```
Comment (senysenyseny16) [inline README.md:237]:
```suggestion
model = onnx.load('model.onnx')
```

File: README.md [hunk 2]
```diff
@@ -218,6 +218,35 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:
...
+# Convert the model to the target version
+target_version = 13
+converted_model = version_converter.convert_version(model, target_version)
+# Convert to torch
+torch_model = convert(converted_model)
+torch.save(torch_model, "model.pt")
```
Comment (senysenyseny16) [inline README.md:243]:
```suggestion
torch.save(torch_model, 'model.pt')
```

**Review by senysenyseny16 — CHANGES_REQUESTED** [review]
Hello, @SuperSecureHuman.

Thanks for your contribution!

Could you fix the code to match our code style?

File: README.md [hunk 3]
```diff
@@ -218,6 +218,35 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:
 Here we have used a trick to convert the model from torch back to ONNX by defining the custom ``_ScatterNDExportToOnnx``.
 </details>
 
+## Opset version workaround
+
+Incase you are using a model with older opset, try the following workaround
```
[the remark this answers is not in the document]
Comment (SuperSecureHuman) [reply README.md]: Added 

https://github.com/ENOT-AutoDL/onnx2torch/pull/130/commits/3063242050f31e0cdc73d60f622fb13791b46f44

**Review by senysenyseny16 — APPROVED** [review]
Thanks!

**Pull request merged**